From 85d7b86fc8170cd065f1329352c8cd30051b9aad Mon Sep 17 00:00:00 2001
From: Olivier Humbert <trebmuh@users.noreply.github.com>
Date: Fri, 18 Nov 2016 16:29:57 +0100
Subject: [PATCH 1088/1936] update copyright

The idea is to update what's displayed when you go to : ZynAddSubFX menu -> File -> Copyright to something more updated/accurate.
I didn't test it. Please check if it's done correctly.
---
 src/UI/MasterUI.fl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/UI/MasterUI.fl b/src/UI/MasterUI.fl
index c942af51..02b3ca01 100644
--- a/src/UI/MasterUI.fl
+++ b/src/UI/MasterUI.fl
@@ -2,7 +2,7 @@
 version 1.0302 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2009 Nasca Octavian Paul} {private local
+decl {//Copyright (c) 2002-2009 Nasca Octavian Paul - (c) 2009-2016 Mark McCurry} {private local
 } 
 
 decl {//License: GNU GPL version 2 or later} {private local
@@ -869,7 +869,7 @@ panelwindow->show();}
       xywh {411 344 365 280} type Double hide
     } {
       Fl_Box {} {
-        label {Copyright (c) 2002-2009 Nasca O. PAUL and others. Please read AUTHORS.txt}
+        label {Copyright (c) 2002-2009 Nasca O. PAUL, 2009-2016 Mark McCurry, and others. Please read AUTHORS.txt}
         xywh {15 35 335 55} labeltype EMBOSSED_LABEL labelsize 15 align 208
       }
       Fl_Box {} {
-- 
2.47.0

